cmd/server_test.go

```diff
@@ -522,32 +522,6 @@ func (s *TestSuiteCommon) TestListenBucketNotificationHandler(c *check) {
 	if s.signer == signerV4 {
 		verifyError(c, response, "XAmzContentSHA256Mismatch", "The provided 'x-amz-content-sha256' header does not match what was computed.", http.StatusBadRequest)
 	}
-
-	// Change global value from 5 second to 100millisecond.
-	globalSNSConnAlive = 100 * time.Millisecond
-	req, err = newTestSignedRequest("GET",
```

Comment:
Why not leave this here for now? @krisis - this can be fixed when the Bala's refactor change is merged.

Reply:
It serves no purpose. It sends a http request to the listenBucketNotication handler and nothing else. So, there is never a bad time to remove it.